src/components/Download/DownloadLink.tsx

```diff
@@ -0,0 +1,38 @@
+import Link from '../Link';
+
+export interface DownloadLinkProps extends Omit<React.HTMLAttributes<HTMLAnchorElement>, 'href' | 'children'> {
+  /**
+   * Optionally display download link as a button
+   * */
+  asButton?: boolean;
```

Comment:
we tend to handle variants as strings (see default vs assertive button for example). can we align with existing patterns?